src/Nix.hs

```diff
@@ -11,6 +11,7 @@ module Nix (module Nix.Cache,
             module Nix.Render.Frame,
             module Nix.Normal,
             module Nix.Options,
+            module Nix.NixString,
```

Comment:
I think, for consistency's sake, that while `NixString` is a fine name for the type, the module is better named just `Nix.String`. `Nix.NixString` seems redundant...